.travis.yml

```diff
@@ -1,12 +1,15 @@
+dist: focal
 language: node_js
 node_js:
-    - 12
+    - lts/*
+    - 18
```

Comment:
Do we list two versions under `node_js` option?

Reply:
yes. we can test on multiple node versions